pyccel/ast/numpyext.py

```diff
@@ -1320,3 +1319,80 @@ class NumpyInt64(NumpyInt):
                 decorators={'property':'property', 'numpy_wrapper':NumpyReal}),
             FunctionDef('diagonal',[],[],body=[],
                 decorators={'numpy_wrapper':NumpyDiag})])
+
+#==============================================================================
+# TODO split numpy_functions into multiple dictionaries following
+# https://docs.scipy.org/doc/numpy-1.15.0/reference/routines.array-creation.html
+numpy_functions = {
+    # ... array creation routines
+    'full'      : NumpyFull,
+    'empty'     : NumpyEmpty,
+    'zeros'     : NumpyZeros,
+    'ones'      : NumpyOnes,
+    'full_like' : NumpyFullLike,
+    'empty_like': NumpyEmptyLike,
+    'zeros_like': NumpyZerosLike,
+    'ones_like' : NumpyOnesLike,
+    'array'     : NumpyArray,
+    # ...
+    'shape'     : Shape,
```

Comment:
Was Shape missed in the last PR?

Reply:
No, It was there.
check [this](https://github.com/pyccel/pyccel/pull/524/files#diff-045bf48321280ff9b7f386d34e81f62aa08ce3e66405adba5f5675d123aeb725L69-L70).

Reply:
I mean why is it not `NumpyShape`

Reply:
I asked myself the same question, and I found the answer.
now I forget.
I will reconsider that.

Reply:
Is it because Shape is a function rather than a class?

Reply:
yes. I thought about that.

Reply:
we should respect name convention